src/conn/conn_compact.c

```diff
@@ -21,6 +21,92 @@ __compact_server_run_chk(WT_SESSION_IMPL *session)
     return (FLD_ISSET(S2C(session)->server_flags, WT_CONN_SERVER_COMPACT));
 }
 
+/*
+ * __background_compact_add_exclude_entry --
+ *     Add the entry to the exclude hash table.
+ */
+static int
+__background_compact_add_exclude_entry(WT_SESSION_IMPL *session, const char *name, size_t len)
+{
+    WT_BACKGROUND_COMPACT_EXCLUDE_ENTRY *new_entry;
+    WT_CONNECTION_IMPL *conn;
+    WT_DECL_RET;
+    uint64_t bucket, hash;
+
+    conn = S2C(session);
+
+    WT_ERR(__wt_calloc_one(session, &new_entry));
+    WT_ERR(__wt_strndup(session, name, len, &new_entry->name));
+
+    hash = __wt_hash_city64(name, len);
+    bucket = hash & (conn->hash_size - 1);
+    /* Insert target uri entry into hashtable. */
+    TAILQ_INSERT_HEAD(&conn->background_compact.compactexcludeentryhash[bucket], new_entry, hashq);
```

Comment:
Some other name for ```compactexcludeentryhash``` ? for instance ```exclude_list_hash```

Reply:
I like this, see cc52c025926072ed826202d782fc9641f879cd2a.